dsp/modules/cohere.py

```diff
@@ -6,10 +6,14 @@
 
 try:
     import cohere
+
     cohere_api_error = cohere.errors.UnauthorizedError
 except ImportError:
     cohere_api_error = Exception
     # print("Not loading Cohere because it is not installed.")
+except AttributeError:
+    cohere_api_error = Exception
```

Comment:
I'm guessing the location of the errors for cohere are different across 4.X.X and 5.X.X. Could have one more fallback for 4.X.X if supporting 4 fully is desired.

Reply:
They actually aren't different, it was just wrong before from what I could dig up